index.html

```diff
@@ -9,51 +9,229 @@
+                    <span class="year">2022</span>
+                </p>
+                
+                <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+            </div>
+    
+            <ul class="programing-languages">
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javaScript</a></li>
+            </ul>
+    
+            <button type="button">See Project</button>
+        </section>
+    </section>
+
+    <!--About Section-->
+    <section class="about">
+        <div class="about-me">
+            <h3 class="intro">About Myself</h3>
+            <p class="main-paragraph">Hello, I'm a software developer! I can help you build a product, feature or website. 
+                Look through some of my work and experience! If you like what you see and have a project you need coded,
+                 don't hesitate to contact me.
+            </p>
+            <p class="connect">LET'S CONNECT</p>
         </div>
-        <nav class="menu">
-            <a href="#">
-                <img src="./images/Icon-Menu.png" alt="Menu" class="menu-icon"/>
-            </a>
-        </nav>
-    </header>
-    <!--Main Section-->
-    <main>
-        <h1 class="intro">I'm Ernest Glad to see you!</h1>
-        <p class="main-paragraph">I am a Web developer! I can help you build your website. Kindly go through my work samples! If you are interested and you have any website to develop, do contact me. I would appreciate recommendation as well.</p>
-        <p class="connect">LET'S CONNECT</p>
-    </main>
-    <!--Headline Section-->
-    <section id="headline-section">
+        
+        <div class="headline-section">
+            <ul>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/twiter.png" alt="Twitter"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social linkendIn">
+                        <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/mail.png" alt="Email"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social github">
+                        <img src="./images/Vector.png" alt="GitHub"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social peace">
+                        <img src="./images/fingers.png" alt="Peace"/>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <button type="button" class="get-resume">Get my resume</button>
```

Comment:
Your about-me section is not in accordance with the Figma template of your choosing. Please update this accordingly.
If you notice, the whole app has a background color that contrasts with the white background of the about-me section. Also, the about-me section has a border radius on the top-right side. Also, change the background of the button.
| Figma template  | Your design |
| ------------- | ------------- |
| ![about](https://user-images.githubusercontent.com/33268927/170699259-e16fbcd0-7cff-4263-9171-5d6757b4777a.png) | ![about-bad](https://user-images.githubusercontent.com/33268927/170699305-2060d20e-89b5-4cbb-9774-d40266849a23.png) |